troubleshooting.md

```diff
@@ -919,3 +919,113 @@ After deleting a client VM on macOS via `podman machine stop` && `podman machine
 ### Solution
 
 You will need to remove the hanging gv-proxy process bound to the port in question. For example, if the port mentioned in the error message is 127.0.0.1:7777, you can use the command `kill -9 $(lsof -i:7777)` in order to identify and remove the hanging process which prevents you from starting a new VM on that default port.
+
+### 32) Passed-in devices or files can't be accessed in rootless container or a container creates files that can't be accessed by the regular user on the host
```

Comment:
These should be broken into two separate examples. Putting them into 1 is confusing to me.

Reply:
You're right. I split them in two.